projects/project-tooling.md

```diff
@@ -0,0 +1,38 @@
+# Project Tooling SIG
+
+## Description
+
+Currently, a variety of tools exist around OpenTelemetry for various
+organizational purposes -- GitHub bots, for example, or the Stack Overflow to
+Slack action. This SIG is being established to create sustaining engineering
+efforts and best practices around these, and new, tools.
+
+## Deliverables
+
+- Document existing tools and their purpose, along with documentation and build
+  instructions.
+- Interview and collect feedback on cross-functional tooling needs (e.g., bots
+  for GitHub issue management, IaC for organization management.)
+- Institute best practices for tooling development and maintenance.
+- Develop new tools as needed.
```

Comment:
```suggestion
- Develop new tools as needed.
- Project-wide service for release and package management
```

See my comment, I would like to establish this "as a service", so SIGs might be able to delegate the package management and focus on other demands.